### PR TITLE
user/plan9port: new package (0_git20240110)

### PR DESCRIPTION
**Pull request by z-erica**

### CODE REVIEW

**Comment by q66** [comment]
the policy is in general not to package stuff that does not have tagged releases (outside of special exceptions), see packaging.md

File: contrib/plan9port/template.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+license = "MIT"
+url = "https://9fans.github.io/plan9port"
+source = f"https://github.com/9fans/plan9port/archive/{_commit}.zip"
+sha256 = "931b96e814b31769bf62b2ef2189ac2f474207d079932987c468b1c86854fb39"
+options = ["!cross"]
+
+
+def do_configure(self):
+    self.cp(self.files_path / "configure", ".")
+    self.do("./configure")
```
Comment (nekopsykose) [inline contrib/plan9port/template.py:26]: you can also write those contents from here instead of needing a script

**Comment by JamiKettunen** [comment]
should be moved under [the newly added `user/`](https://github.com/chimera-linux/cports/compare/c307559...383095d) after rebase

File: user/plan9port/files/acme [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec 9 acme
```
Comment (nekopsykose) [inline user/plan9port/files/acme:2]: does this need "$@" too? (and i'm not sure if it accepts files passed, for perhaps %U or whatnot to the desktop entry Exec (see the spec))

File: user/plan9port/template.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+url = "https://9fans.github.io/plan9port"
+source = f"https://github.com/9fans/plan9port/archive/{_commit}.zip"
+sha256 = "931b96e814b31769bf62b2ef2189ac2f474207d079932987c468b1c86854fb39"
+options = ["!cross", "!lintstatic"]
+
+
+def do_configure(self):
+    with open(self.cwd / "LOCAL.config", "w") as config:
+        config.write(f"CC9=" + self.get_tool("CC") + "\n")
+        config.write(f"CC9FLAGS='" + self.get_cflags(shell=True) + "'\n")
```
Comment (nekopsykose) [inline user/plan9port/template.py:26]: ah missed this before- ldflags are ignored too, so you want to put them in this same cc9flags array i guess (gives a bunch of warnings but there's no other way that i see)

**Pull request merged**